metadata.json

```diff
@@ -5,6 +5,6 @@
   "url": "https://github.com/josephlbarnett/autoselectheadset-gnome-shell-extension",
   "version": "9",
   "shell-version": [
-    "45"
```

Comment:
i assume we want `"45","46"`, or did something change in gnome-shell where that's no longer needed?

Reply:
Agree. That would also include backward compatibility.